retriever/lib/install.py

```diff
@@ -1,209 +0,0 @@
-from __future__ import print_function
-from __future__ import absolute_import
-import imp
-import os
-import sys
-from pkg_resources import parse_version
-from retriever.engines import choose_engine
-from retriever.lib.compile import compile_json
-from retriever.lib.defaults import DATA_DIR, SCRIPT_SEARCH_PATHS, VERSION
-from retriever.lib.tools import name_matches
-from retriever.lib.scripts import SCRIPT_LIST
-
-script_list = SCRIPT_LIST()
-
-
-def _compile(dataset):
-    json_file = dataset + '.json'
-    py_file = dataset + '.py'
-
-    for search_path in [search_path for search_path in SCRIPT_SEARCH_PATHS if os.path.exists(search_path)]:
-        files = os.listdir(search_path)
-        for file in files:
-            if file == json_file:
-                compile_json(os.path.join(search_path, dataset))
-            if file == py_file:
-                file, pathname, desc = imp.find_module(dataset, [search_path])
-                try:
-                    new_module = imp.load_module(dataset, file, pathname, desc)
-                    if hasattr(new_module.SCRIPT, "retriever_minimum_version"):
-                        # a script with retriever_minimum_version should be loaded
-                        # only if its compliant with the version of the retriever
-                        if not parse_version(VERSION) >= parse_version("{}".format(
-                                new_module.SCRIPT.retriever_minimum_version)):
-                            print("{} is supported by Retriever version {}".format(dataset,
-                                                                                   new_module.SCRIPT.retriever_minimum_version))
-                            print("Current version is {}".format(VERSION))
-                            continue
-                    # if the script wasn't found in an early search path
-                    # make sure it works and then add it
-                    new_module.SCRIPT.download
-                except Exception as e:
-                    sys.stderr.write("Failed to load script: %s (%s)\nException: %s \n" % (
-                        dataset, search_path, str(e)))
-
-
-def _install(args, use_cache, debug, compile):
-    """Calling the download-install function."""
-    engine = choose_engine(args)
-    engine.use_cache = use_cache
-
-    if compile:
```

Comment:
We do not need two function that look the same.  lets not worry about compiling specific scripts because we are still compiling the entire scripts at all times.